events/wc_dark_cult_events.txt

```diff
@@ -1101,11 +1150,7 @@ narrative_event = {
 			add_society_currency_massive_effect = yes # Give the Grand Master a start-up capital to induct members
 		}
 
-		add_35_void_corruption_effect = yes
-
 		hidden_effect = {
-			activate_title = { title = d_twilights_hammer status = no }
-			d_twilights_hammer = { destroy_landed_title = THIS }
```

Comment:
Hate it, why this pitiful ogre rules my Twilight's Hammer.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79171196-7c613080-7e02-11ea-843f-21d6032a5c5e.png)

</details>

Reply:
well society and clan are two different things plus the clan is destroyed if cho'gall decides to not go old gods

Reply:
> well society and clan are two different things plus the clan is destroyed if cho'gall decides to not go old gods

On my screenshot, Cho'gall decided not to go old gods

Reply:
event `WCDAC.5005` destroys the title when he does it

Reply:
> event `WCDAC.5005` destroys the title when he does it

On my screenshot, it's not destroyed.

Reply:
well it works ok for me